monitoring/rid_qualifier/aircraft_state_replayer.py

```diff
@@ -0,0 +1,151 @@
+from monitoring.monitorlib.auth import make_auth_adapter
+from monitoring.monitorlib.infrastructure import DSSTestSession
+import asyncio
+from monitoring.monitorlib import rid
+import json, os
+import uuid
+from pathlib import Path
+from typing import  Any
+from monitoring.rid_qualifier.utils import OperatorLocation, RIDFlightDetails, TestFlightDetails, TestFlight, TestPayload, DeliverablePayload
+import arrow
+from uuid import UUID
+from typing import List
+from monitoring.rid_qualifier.utils import RIDQualifierTestConfiguration
+
+class TestBuilder():
+    ''' A class to setup the test data and create the objects ready to be submitted to the test harness '''
+
+    def __init__(self, test_configuration: RIDQualifierTestConfiguration) -> None:
+        self.test_configuration = test_configuration
+
+        self.aircraft_states_directory = Path('test_definitions', test_configuration.locale, 'aircraft_states')
+        
+        self.verify_aircraft_states_directory(self.aircraft_states_directory)
+        self.aircraft_state_files = os.listdir(self.aircraft_states_directory) 
+                
+        self.usses = self.test_configuration.usses
```

Comment:
No need to store this separately as a new property of the class instance; it is already stored as part of `self.test_configuration`.  A local variable would be fine instead.

Reply:
Thanks! Yes now it is done!